albumentations/core/composition.py

```diff
@@ -474,3 +474,15 @@ def _to_dict(self):
             }
         )
         return data
+
+
+class Sequential(BaseCompose):
+    """Sequentially applies all transforms to targets."""
+
+    def __init__(self, transforms, p=0.5):
+        super().__init__(transforms, p)
+
+    def __call__(self, **data):
+        for t in self.transforms:
+            data = t(**data)
+        return data
```

Comment:
I am not sure that `DualTransforms` will work correctly with this wrapper, because we are do not convert bboxes and keypoints to our format.
Also this call doesn't contain preprocessing and postprocessing

Reply:
Not sure if I understood you correctly. `Sequential` is not intended to be a replacement for `Compose`. Instead, it should be used inside `Compose` the same way `OneOf` or `OneOrOther` are used now. 

The main use case for AutoAlbument is the following: we have N sub-policies, each sub-policy contains M augmentations.
We want to choose one sub-policy at random and apply its augmentations in a sequence. With `Sequential` we can implement this pipeline. For example, here is a definition if a pipeline that contains 3 sub-policies and each subpolicy has 2 sequential augmentations:

```
transform = A.Compose([
    A.OneOf([
        A.Sequential([
            A.HorizontalFlip(p=0.5),
            A.ShiftScaleRotate(p=0.5),
        ]),
        A.Sequential([
            A.VerticalFlip(p=0.5),
            A.RandomBrightnessContrast(p=0.5),
        ]),
        A.Sequential([
            A.RGBShift(p=0.5),
            A.Cutout(p=0.5),
        ])
    ], p=1)
])
```

Reply:
I think we need write this tips in `Sequentional` documentation, that it is must be inside `Compose` and can not work alone.

Reply:
Done